README.md

```diff
@@ -22,4 +23,8 @@ Templates, random tables, character sheets and other stuff for playing solo RPGs
 - Homepage.
 - Timelines.
 - Tag Wrangler.
-- Gallery.
\ No newline at end of file
+- Gallery.
+
+## Extras
+
+- [Publish to Github Pages](https://github.com/spacegiant/obsidian-github-pages-starter)
```

Comment:
Why do you link to this?

Reply:
This was requested addition on discord. This is instructions on how to automate publishing your obsidian actual play notes to a GitHub pages website.

Reply:
Feel free to add the page directly if that makes more sense